### ISSUE TITLE
Tp code dashboard 

### ISSUE THREAD

**Issue by Shauryan123** [nus-cs2103-AY2324S2/forum#579]
if u create a seperate branch in the team repo … because u r implementing a feature and make PRS from
Ur own fork to that branch u newly created (not master) … and then merge it into the master branch of team repo once the feature is done ..will the LoC u had done while making PRs to the new branch be reflected in the code dashboard once u merge into master ?

**Comment by damithc** [comment]
@Shauryan123 Yes.
